app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -33,27 +33,36 @@ def proof(
           user_email: user_email,
         )
 
-        # todo(LG-8693): Begin verifying both the user's residential address and identity document
-        # address
+        residential_instant_verify_proof = proof_residential_address_if_needed(
+          applicant_pii: applicant_pii,
+          timer: timer,
+          double_address_verification: double_address_verification,
+        )
+
         applicant_pii = with_state_id_address(applicant_pii) if double_address_verification
 
-        resolution_result = proof_resolution(
+        instant_verify_proof = proof_id_address_with_lexis_nexis_if_needed(
```

Comment:
I did renaming in the file to try and make it clearer, eg, changing `resolution_result` to `instant_verify_proof`.  I would be interested in people's thoughts on these changes, please.

Reply:
👍 i found that name is confusing, this clears it up.

Reply:
Yeah, I like anything that makes it clear which vender is doing what, so this is good!

Reply:
it's storing the result correct? can we still keep the word result in the name, I think of proof as evidence. So maybe something like `instant_verify_proof_result`?

Reply:
Good point, Shannon.  I changed it from `instant_verify_proof ` to `instant_verify_result`.